src/account/account.cairo

```diff
@@ -71,9 +70,9 @@ mod AccountComponent {
 
             // Check tx version
             let tx_info = get_tx_info().unbox();
-            let version = tx_info.version;
-            if version != TRANSACTION_VERSION {
-                assert(version == QUERY_VERSION, Errors::INVALID_TX_VERSION);
+            if (tx_info.version != QUERY_VERSION) {
```

Comment:
i think should follow the same `<=` logic, to support v3 simulations. it should also be tested.